CMakeLists.txt

```diff
@@ -6,3 +6,5 @@ set(CMAKE_CXX_STANDARD 17)
 
 add_subdirectory(external)
 add_subdirectory(src)
+
+include_directories(include)
```

Comment:
I put headers in include/ folder, figured this should be added here (I might be wrong though, never used this)